code-samples/serving/hello-world/helloworld-java-spring/README.md

```diff
@@ -133,101 +133,100 @@ During the creation of a Service, Knative performs the following steps:
 
 To deploy the app:
 
-=== "YAML"
-
-    1. Create a new file named `service.yaml` and copy the following service definition
-    into the file:
-
-        ```yaml
-        apiVersion: serving.knative.dev/v1
-        kind: Service
-        metadata:
-          name: helloworld-java-spring
-          namespace: default
-        spec:
-          template:
-            spec:
-              containers:
-                - image: docker.io/{username}/helloworld-java-spring
-                  env:
-                    - name: TARGET
-                      value: "Spring Boot Sample v1"
-        ```
-        Where `{username}` is your Docker Hub username.
-
-        **Note:** Ensure that the container image value in `service.yaml` matches the container you built in the previous step.
+### YAML
+
+1. Create a new file named `service.yaml` and copy the following service definition
+   into the file:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-java-spring
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: docker.io/{username}/helloworld-java-spring
+          env:
+            - name: TARGET
+              value: "Spring Boot Sample v1"
+```
+Where `{username}` is your Docker Hub username.
```

Comment:
```suggestion
    ```yaml
    apiVersion: serving.knative.dev/v1
    kind: Service
    metadata:
      name: helloworld-java-spring
      namespace: default
    spec:
      template:
        spec:
          containers:
            - image: docker.io/{username}/helloworld-java-spring
              env:
                - name: TARGET
                  value: "Spring Boot Sample v1"
    ```
    Where `{username}` is your Docker Hub username.
```

Reply:
Indentation required to make the numbering format properly. As you can see in [this preview](https://github.com/knative/docs/blob/bf85d69a192ed556c1e8956ea1b22ecbe341fdcd/code-samples/serving/hello-world/helloworld-java-spring/README.md#yaml), the numbering is broken.